util/cleanup.h

```diff
@@ -36,6 +36,13 @@ static inline void close_file(int *f)
 }
 #define _cleanup_file_ __cleanup__(close_file)
 
+static inline void close_stream(FILE **f)
+{
+	if (*f)
+		fclose(*f);
+}
+#define _cleanup_stream_ __cleanup__(close_stream)
```

Comment:
after the cleanup.h sync in https://github.com/linux-nvme/nvme-cli/pull/2395 this should be 
```
static inline DEFINE_CLEANUP_FUNC(cleanup_file, FILE *, fclose)
#define _cleanup_file_ __cleanup__(cleanup_file)
```